client/src/app/core/ui-services/linenumbering.service.ts

```diff
@@ -244,15 +246,9 @@ export class LinenumberingService {
      * @return {DocumentFragment}
      */
     private htmlToFragment(html: string): DocumentFragment {
-        const fragment: DocumentFragment = document.createDocumentFragment(),
-            div = document.createElement('DIV');
-        div.innerHTML = html;
-        while (div.childElementCount) {
-            const child = div.childNodes[0];
-            div.removeChild(child);
-            fragment.appendChild(child);
-        }
-        return fragment;
+        const template = document.createElement('template');
+        template.innerHTML = html;
+        return template.content;
```

Comment:
the exact same function was declared twice. Perhaps create a service for shared document functions.